tests/ast/data/functions.test

```diff
@@ -656,9 +656,9 @@ df27 = df.select(bitshiftright(col("A"), col("B")), bitshiftright(col("A"), -10)
 
 df28 = df.select(bround("A", 10), bround("A", 2), bround(col("A"), col("B")))
 
-df29 = df.select(convert_timezone(col("A"), col("B")), convert_timezone(col("A"), col("B")), convert_timezone(col("A"), col("B")), convert_timezone(col("A"), col("B")), convert_timezone(col("A"), col("B")), convert_timezone(col("A"), col("A"), col("B")))
+df29 = df.select(convert_timezone("A", col("B")), convert_timezone(col("A"), "B"), convert_timezone("A", "B"), convert_timezone(col("A"), col("B")), convert_timezone("A", "B"), convert_timezone("A", "B", "A"))
```

Comment:
Why should col() be removed here?  Does it no longer apply to a column reference?

Reply:
So, if you're looking at the original code run:
```py
df29 = df.select(convert_timezone("A", col("B")), convert_timezone(col("A"), "B"), convert_timezone("A", "B"), convert_timezone(col("A"), col("B")), convert_timezone("A","B",None), convert_timezone("A", "B", "A"))
```
"A" wasn't passed in as a column -- I changed the AST encoder to record what the actual parameter passed in is as opposed to what parameter was given to `builtin`.